packages/mco/components/mco-dashboard/data-policy/dr-dashboard.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { useCustomPrometheusPoll } from '@odf/shared/hooks/custom-prometheus-poll';
+import { Grid, GridItem } from '@patternfly/react-core';
+import { ACM_ENDPOINT, HUB_CLUSTER_NAME } from '../../../constants';
+import { useArgoApplicationSetResourceWatch } from '../../../hooks';
+import { StorageDashboard, STATUS_QUERIES } from '../queries';
+import { AlertsCard } from './alert-card/alert-card';
+import { ClustersCard } from './cluster-app-card/cluster-card';
+import {
+  CSVStatusesContext,
+  DRResourcesContext,
+} from './policy-dashboard-context';
+import { StatusCard } from './status-card/status-card';
+import { SummaryCard } from './summary-card/summary-card';
+import '../../../style.scss';
+
+const UpperSection: React.FC = () => (
+  <Grid hasGutter>
+    <GridItem lg={8} rowSpan={3} sm={12}>
+      <StatusCard />
+    </GridItem>
+    <GridItem lg={8} rowSpan={3} sm={12}>
+      <SummaryCard />
+    </GridItem>
+    <GridItem lg={4} rowSpan={6} sm={12}>
+      <AlertsCard />
+    </GridItem>
+    <GridItem lg={12} rowSpan={6} sm={12}>
+      <ClustersCard />
+    </GridItem>
+  </Grid>
+);
+
+export const MCODashboardPage: React.FC = () => {
```

Comment:
```suggestion
export const DRDashboard: React.FC = () => {
```

under MCO, we have multiple dashboards like storage system, DR dashboard, etc..

Reply:
yeah missed ur earlier comment...

Reply:
updated...